src/components/App/App.js

```diff
@@ -23,6 +44,27 @@ class App extends Component {
     }
   }
   render() {
+    // working
+    // getAllRecipes()
+    // getUserWithRecipes(1)
+    // boughtRecipesByUser(1)
+    // recipeById(1)
+    // getUser(1)
+    // searchNonProfits("Food Bank of the Rockies")
+    // createUserRecipe(1, 3, 2.5)
+    // createIngredient(1, "name", "amount")
+    // updateUserRecipeRating(2, 4) working but I found a bug in BE
+    // registerUser("John", "Doe", "JD@gmail.com", "a street", "a city", "a state", "a zip", "an image", "new user", "1234")
+    // userSignIn("new user", "1234")
+
+    // parse errors (coming back to this one later, I cant figure out how to pass the ingredients in a way it likes)
+    // createRecipe(1, "Chicken Parmesan", "A classic favorite", "1. chicken 2.???? 3. profit", 533423, "Give Logan Money", {name: "Chicken", amount: "2 lbs"}, {name: "Parmesan", amount: "5 gallons"})
+
+    // Must have form fields autopopulated in order for these to work
+    // updateUser()
+    // updateIngredient()
+    // updateRecipe() (Does not exist on BE yet)
+
     return (
```

Comment:
Sorry for the mess here, but it's the only way I could verify the code was working

Reply:
No worries at all!
It'll be helpful in the build process for sure!